oceannavigator/frontend/src/components/lib/Accordion.jsx

```diff
@@ -0,0 +1,52 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Panel } from "react-bootstrap";
+
+import Icon from "./Icon";
+import { withTranslation } from "react-i18next";
+
+export class Accordion extends React.Component { 
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      open: false,
+      panelIcon: "angle-down"
+    };
+  };
+
+  render() {
+    return (
+      <Panel 
+        id={this.props.id}
+        expanded={this.state.open}
+        onToggle={}
```

Comment:
Necessary to suppress an error from React. See https://github.com/react-bootstrap/react-bootstrap/issues/3141.